spec/triggers/new_event_spec.rb

```diff
@@ -0,0 +1,267 @@
+# rubocop:disable Metrics/BlockLength, Layout/LineLength
+# frozen_string_literal: true
+
+RSpec.describe 'triggers/new_event', :vcr do
+  # Spec describes the most commons blocks of a trigger.
+  # Depending on the type of your trigger remove describes that you don't need.
+  # Learn more: https://docs.workato.com/developing-connectors/sdk/cli/reference/rspec-commands.html
+
+  let(:connector) { Workato::Connector::Sdk::Connector.from_file('connector.rb', settings) }
+  let(:settings) { Workato::Connector::Sdk::Settings.from_encrypted_file('settings.yaml.enc', 'master.key') }
+
+  # Or add more fine grained tests for each trigger definition block
+  let(:trigger) { connector.triggers.new_event }
+
+  describe 'Run entire trigger with valid input' do
+    subject(:output) { connector.triggers.new_event(input) }
+    let(:input) { JSON.parse(File.read('fixtures/triggers/input/webhook.json')) }
+
+    it 'person.create' do
+      # TODO
```

Comment:
Add body?

Reply:
Removed it, as there is no body. Will be in the next commit.